foreman.toml

```
@@ -1,4 +1,4 @@
 [tools]
 rojo = { source = "rojo-rbx/rojo", version = "7.1.1" }
 run-in-roblox = { source = "rojo-rbx/run-in-roblox", version = "0.3.0" }
-selene = { source = "Kampfkarren/selene", version = "0.17.0" }
+selene = { source = "Kampfkarren/selene", version = "0.18.1" }
```

Upgrade to [0.18.2](https://github.com/Kampfkarren/selene/releases/tag/0.18.2):
```suggestion
selene = { source = "Kampfkarren/selene", version = "0.18.2" }
```